src/shared/bpf-dlopen.c

```diff
@@ -181,6 +181,19 @@ int dlopen_bpf(void) {
         return r;
 }
 
+int bpf_get_error_translated(const void *ptr) {
```

Comment:
seems like a candidate for static inline, but i don't care much

Reply:
ah, you don't want this, so that the original symbol doesn't have to be exported? I guess that's a valid goal. ignore me then.

Reply:
yeah so that it doesn't get used accidentally